control_panel_api/serializers.py

```diff
@@ -6,7 +6,8 @@
     AppS3Bucket,
     S3Bucket,
     User,
-    UserS3Bucket)
+    UserS3Bucket,
+    AppUser)
```

Comment:
Minor style point, I tend to keep these sorted alphabetically and also leave a trailing comma/newline after last element, e.g.

```python
from control_panel_api.models import (
    AppS3Bucket,
    # ...
    UserApp,
    UserS3Bucket,
)
```

...but this is really me being a bit of an OCD person, sorry :P

Reply:
+1

Reply:
Not that bothered about the trailing comma, though - it's a bad habit to get into if you then start writing JSON, where it is an error

Reply:
Agreed, I like the trailing commas as then someone else can add an element at the end without modifying the existing lines of code. But yes, not in JSON as not valid 🐛

Reply:
Although it's a tiny thing, I apply YAGNI - code for what you need now, not what you might need in future.

Reply:
I like the open-close principle although: https://en.wikipedia.org/wiki/Open/closed_principle

> software entities (classes, modules, functions, etc.) should be open for extension, but closed for modification

e.g. the next person can add a line to the list without modifying existing lines - but as you said, it's a tiny application of it. Just to be clear, I'm not that bothered really :)

Reply:
The open-closed principle is a good one, but it doesn't apply to lists in source code 😛